<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrMpixRead</class>
 <widget class="QMainWindow" name="PrMpixRead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrMpixRead</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="settings_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>381</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="InfooutTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>351</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="Button_graph">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_lineedit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X max</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N bins</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_xmin"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_xmax"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_nbins"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>X min</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zooming</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_HORIZ">
          <property name="text">
           <string>Horiz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_VERT">
          <property name="text">
           <string>Vert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_RECT">
          <property name="text">
           <string>Rect</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="scrool_frame">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>44</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>44</width>
       <height>521</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QScrollBar" name="verScrollB_down">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
        <property name="invertedControls">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSpinBox" name="spinBox_down">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QSpinBox" name="spinBox_up">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScrollBar" name="verScrollB_up">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
        <property name="invertedControls">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="graph_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>541</width>
      <height>491</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File ...</string>
    </property>
    <addaction name="actionOpen_pixel"/>
    <addaction name="actionOpen_mask"/>
    <addaction name="actionSave_canv_As"/>
    <addaction name="separator"/>
    <addaction name="actionFlip_matrix_of_mask"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
    <addaction name="actionSpectrogramma_of_Working_Pixels"/>
    <addaction name="actionSpectrogramma_of_Masking_Pixels"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_pixel">
   <property name="text">
    <string>Open 256x256 pixel</string>
   </property>
  </action>
  <action name="actionOpen_mask">
   <property name="text">
    <string>Open 256x256 mask</string>
   </property>
  </action>
  <action name="actionSave_canv_As">
   <property name="text">
    <string>Save canv As </string>
   </property>
  </action>
  <action name="actionSpectrogramma_of_Working_Pixels">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrogramma of Working Pixels</string>
   </property>
  </action>
  <action name="actionSpectrogramma_of_Masking_Pixels">
   <property name="text">
    <string>Spectrogramma of Masking Pixels</string>
   </property>
  </action>
  <action name="actionFlip_matrix_of_mask">
   <property name="text">
    <string>Flip matrix of mask</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_down</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verScrollB_down</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_up</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verScrollB_up</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verScrollB_up</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_up</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verScrollB_down</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_down</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
